test/configCases/plugins/uglifyjs-plugin/extract.js

```diff
@@ -1,7 +1,9 @@
-/** @preserve comment should be extracted extract-test.1 */
-
 var foo = {};
 
+// ⚠️ move the following comment back to the top
```

Comment:
Is this still relevant?

Reply:
Yep, but it's seems to be a known upstream issue in `uglify-es`